createtransfers/createtransfers.py

```diff
@@ -84,16 +84,21 @@ def create_file_and_write(file_name, file_path, encoding):
     file_path_bytes = os.path.join(file_path, file_name_bytes)
     try:
         with open(file_path_bytes, 'w') as fout:
-            fout.write('The name of this file is encoded with encoding'
-                       ' {}\n'.format(encoding))
+            msg = (u'The name of this file is encoded with {} '
+                   'encoding.\n'.format(encoding))
+            if dirs:
+                msg = (u'Part of the directory structure that this '
+                       'file sits within has been encoded with {} encoding.'
+                       '\n'.format(encoding))
+            fout.write(msg)
             LOGGER.info("Created: %s", file_path_bytes)
     except (OSError, IOError) as err:
         ret = "Failed to open file: {0}, error: {1}".format(file_path_bytes,
                                                             err)
         raise CreateTransferException(ret)
 
 
-def rm_dirs_and_create(dir_path):
+def rm_dirs_and_create(dir_path, remove_only=False):
```

Comment:
It seems confusing to have a function named `rm_dirs_and_create` and then have the `rm` part be optional. Can we instead create a new `rm_dirs` func that `rm_dirs_and_create` calls and then have `create_zip_dance` call `rm_dirs`?

Reply:
Makes sense.